OpenRA.Mods.Cnc/Traits/PortableChrono.cs

```diff
@@ -90,17 +92,24 @@ public Order IssueOrder(Actor self, IOrderTargeter order, Target target, bool qu
 			if (order.OrderID == "PortableChronoTeleport")
 				return new Order(order.OrderID, self, queued) { TargetLocation = self.World.Map.CellContaining(target.CenterPosition) };
 
-			return new Order(order.OrderID, self, queued) { TargetActor = target.Actor };
+			return null;
 		}
 
 		public void ResolveOrder(Actor self, Order order)
 		{
-			if (order.OrderString == "PortableChronoTeleport" && CanTeleport)
+			if (order.OrderString != "PortableChronoTeleport")
+				return;
+
+			// Teleport as close as possible
+			if (CanTeleport)
 			{
 				var maxDistance = Info.HasDistanceLimit ? Info.MaxDistance : (int?)null;
 				self.CancelActivity();
 				self.QueueActivity(new Teleport(self, order.TargetLocation, maxDistance, Info.KillCargo, Info.FlashScreen, Info.ChronoshiftSound));
 			}
+
+			// Move normally the rest of the way
+			move.ResolveOrder(self, new Order("Move", self, true) { TargetLocation = order.TargetLocation });
```

Comment:
According to ctor, move can be null. Can it happen in practice? Then a null check is misisng.